src/Button.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+// eslint-disable-next-line react/prop-types
+export default function Button({ children, onClick }) {
+  return (
+    <button type="button" onClick={onClick}>{ children }</button>
```

Comment:
```suggestion
    <button 
      type="button" 
      onClick={onClick}
    >
      { children }
    </button>
```

비교 했을 때 어떤게 더 나은가요?
저는 props 가 2개 이상일 경우 줄바꿈을 해주는 편이에요.
비교해보시고 자신만의 코드 정리 기준을 만들어보시면 좋아요!